bundles/org.openhab.binding.tradfri/pom.xml

```diff
@@ -17,12 +17,12 @@
     <dependency>
       <groupId>org.eclipse.californium</groupId>
       <artifactId>californium-core</artifactId>
-      <version>1.0.7</version>
+      <version>2.0.0-RC1</version>
     </dependency>
     <dependency>
```

Comment:
Travis is failing right now. It complains about a missing dependency. You maybe have to add the `element-connector` to the POM as well.

Reply:
That's interesting. It was not needed before and the code-change does not (directly) require it either. But anyway, now with the merged branch in core, it was easy to validate locally.